ush/forecast_predet.sh

```diff
@@ -122,7 +122,12 @@ FV3_GFS_predet(){
   #file and the value of npe_node_fcst is not correctly defined when using more than
   #one thread and sets NTHREADS_FV3=1 even when the number of threads is appropraitely >1
   #NTHREADS_FV3=${NTHREADS_FV3:-${NTHREADS_FCST:-${nth_fv3:-1}}}
-  NTHREADS_FV3=${nth_fv3:-1}
+  #Threading can be different for GFS and GDAS forecasts
```

Comment:
This should be set in the env file with the others. This whole block can go, as NTHREADS_FV3 is already set there too. I don't know why someone added it back in here, contrary to the comment directly above.

Reply:
OK, I've deleted the block and retested on S4.

Reply:
You need to add a new entry to the block in the env files you moved up to set nth_fv3=nth_fv3_gfs as well.

Reply:
I've added the nth_fv3_gfs assignment to all of the env files.  Thanks for the catch.